.github/workflows/build.yml

```diff
@@ -21,28 +21,24 @@ jobs:
         uses: cisagov/setup-env-github-action@develop
       - uses: actions/checkout@v3
       - id: setup-python
-        uses: actions/setup-python@v3
+        uses: actions/setup-python@v4
```

Comment:
Please update the other use of `actions/setup-python` https://github.com/cisagov/skeleton-ansible-role/blob/945ee638f732cba8506cfeb1c0abfa3ab5993f15/.github/workflows/build.yml#L111

Reply:
Please see commit 1963120.